llvm/utils/TableGen/Common/CodeGenRegisters.cpp

```diff
@@ -2185,11 +2170,13 @@ void CodeGenRegBank::computeRegUnitSets() {
 
       // Find an existing RegUnitSet, or add the union to the unique sets.
       if (findRegUnitSet(RegUnitSets, RUSet) == RegUnitSets.end()) {
-        LLVM_DEBUG(dbgs() << "UnitSet " << RegUnitSets.size() << " "
-                          << RUSet.Name << ":";
-                   for (auto &U
-                        : RUSet.Units) printRegUnitName(U);
-                   dbgs() << "\n";);
+        LLVM_DEBUG({
+          dbgs() << "UnitSet " << RegUnitSets.size() << " " << RUSet.Name
+                 << ":";
+          for (auto &U : RUSet.Units)
+            printRegUnitName(U);
```

Comment:
Use `formatv` or `llvm::interleaved` to eliminate loop? 
But if this stays as is, don't "always auto".

Reply:
moved to helper function.